integr8ly/integreatly-operator
# MGDAPI-3314 update pkgman and create olm/bundle command

# Issue link
https://issues.redhat.com/browse/MGDAPI-3314

# What
Added:
- dockerfiles for rhmi and rhoam to allow developers to quickly and easily build a rhoam/rhmi bundle
- makefile command to support new dockerfile
- missing envs from RHOAM 1.15.1 and 1.15.2
Removed:
- unnecessary bundles from packageman folder

# Verification steps
From this branch run:
```
 make olm/bundle BUNDLE_TAG="quay.io/mstoklus/integreatly-bundle:1.17.0" VERSION=1.15.2 OLM_TYPE=managed-api-service
 ```
 and confirm that the bundle image was built
 ```
  make olm/bundle BUNDLE_TAG="quay.io/mstoklus/integreatly-bundle:1.17.0" VERSION=2.9.0 OLM_TYPE=integreatly-operator
  ```
and confirm rhmi bundle got built
